system-probe_arm64/Dockerfile

```diff
@@ -48,7 +48,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends apt-utils && ap
         python3-dev \
         default-jre \
         wget \
-        xz-utils
+        xz-utils \
+        openssh-client \
```

Comment:
nit: I think the package list was intended to be alphabetical. That way it is easier to see if a package is already included.